app/src/main/java/com/example/henzoshimada/feeltrip/MoodAdapter.java

```diff
@@ -87,7 +108,38 @@ public View getView(int position, View convertView, ViewGroup parent) {
         viewHolder.userName.setText(mood.getUsername());
         viewHolder.date.setText(mood.getDate().toString());
         viewHolder.description.setText(Html.fromHtml(mood.getDescription())); //TODO: This is depreciated, maybe replace?
-        viewHolder.append.setText(" - Feeling " + mood.getEmotionalState());
+        viewHolder.append.setText(" - Feeling ");
+
+        viewHolder.emotion.setText(mood.getEmotionalState());
+        switch (mood.getEmotionalState()) {
+            case "Angry":
+                viewHolder.emotion.setTextColor(Color.RED);
+                break;
+            case "Confused":
```

Comment:
Color coding the emotions, new emotions that we haven't colored yet are set to BLACK by default.